terraform/agent-monitor-module/main.tf

```diff
@@ -114,7 +114,7 @@ resource "elasticsearch_opensearch_monitor" "agent_monitor" {
                     "name": "Notify Automation Queue",
                     "destination_id": "${var.automation_destination_id}",
                     "message_template": {
-                        "source": "{ \"type\": \"agent_down\", \"server\": \"${var.agent_monitor.server}\", \"agent\": \"${var.agent_monitor.agent}\" }",
+                        "source": "{ \"type\": \"agent_down\", \"server\": \"${var.agent_monitor.server}\", \"agent\": \"${var.agent_monitor.agent}\", \"timestamp\": \"Period from {{ctx.periodStart}} to {{ctx.periodEnd}}\" }",
```

Comment:
Maybe just put them both in as 'periodStart' and 'periodEnd'.

Reply:
separated into two